mevislab.github.io/content/tutorials/shorts.md

```diff
@@ -47,3 +48,63 @@ Open {{< menuitem "View" "Views" "Scripting Assistant">}}. A new panel is shown.
 If you now interact with a network, module or macro module, your user interactions are converted into Python calls. You can see the calls in the panel of the Scripting Assistant and copy and paste them for your Python script.
 
 ![Scripting Assistant](/images/tutorials/ScriptingAssistant_Panel.png "Scripting Assistant")
+
+## User Scripts {#user_scripts}
+
+User scripts allow you to call any Python code from the main menu entry {{< menuitem "Scripting">}}. MeVisLab already comes with some user scripts you can try. You can also view the sources for example code via right-click {{< mousebutton "right" >}} on the menu entry under {{< menuitem "Scripting">}}.
+
+This example shows you how to change the color of the MeVisLab IDE to a dark mode.
+
+Right-click {{< mousebutton "right" >}} menu entry {{< menuitem "Scripting" "Utilities" "Close Unselected Panels">}} and select *Edit User Script*. The Python file opens in *MATE*. Right-click {{< mousebutton "right" >}} on the tab in editor and select *Show Enclosing Folder*.
+
+The opened directory contains all available user scripts. Add a new file *MyScripts.def* and open the file in *MATE*.
+
+Enter the following:
+{{< highlight filename="MyScripts.def" >}}
+```Stan
+UserIDEActions {
+  Action "Set Dark Theme" {
+    name       = changeTheme
+    userScript = $(LOCAL)/changeTheme.py
+    statusTip  = "Change Theme to dark mode."
+    accel      = "ctrl+F9"
+  }
+
+}
+
+UserIDEMenus {
+  SubMenu "Theme" {
+    ActionReference = changeTheme
+  }
+}
+```
+{{</highlight>}}
+
+We define an action *Set Dark Theme* added to the submenu *Theme* in MeVisLab IDE menu item {{< menuitem "Scripting">}}. The action is named *changeTheme* and a reference to a Python script is added as *$(LOCAL)/changeTheme.py*. We also defined a keyboard shortcut {{< keyboard "ctrl+F9" >}}.
```

Comment:
added -> , which is added
in MeVisLab IDE menu item -> in the MeVisLab IDE menu item